src/uu/dd/src/parseargs.rs

```diff
@@ -37,41 +37,28 @@ pub enum ParseError {
 }
 
 /// Contains a temporary state during parsing of the arguments
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Default)]
 pub struct Parser {
     infile: Option<String>,
     outfile: Option<String>,
-    ibs: usize,
-    obs: usize,
+    /// The block size option specified on the command-line, if any.
+    bs: Option<usize>,
+    /// The input block size option specified on the command-line, if any.
+    ibs: Option<usize>,
+    /// The output block size option specified on the command-line, if any.
+    obs: Option<usize>,
     cbs: Option<usize>,
     skip: Num,
     seek: Num,
     count: Option<Num>,
     conv: ConvFlags,
+    /// Whether a data-transforming `conv` option has been specified.
+    is_conv_specified: bool,
```

Comment:
Would it make sense to make `conv: Option<ConvFlags>` instead of having a separate flag?

Reply:
Let me try that, I don't remember why I did it this way :thinking:

Reply:
Ah I remember now. It's because there is a bunch of validation and processing code that assumes `conv` exists and has all its fields set to `false` if `conv=...` options are specified on the command line. (See the `Parser::validate()` function.) And I didn't want to go through the hassle of updating all that.

I can update it here though, it seems odd to have this extra flag lying around.